packages/web/utils/compare.ts

```diff
@@ -47,7 +47,8 @@ export function compareDec(a: Dec, b: Dec): CompareResult {
   return 0;
 }
 
-/** Prefers an object that has a given member defined and non-null. */
+/** Prefers an object that has a given member defined and non-null.
+ *  **DOES NOT** compare the value. */
 export function compareDefinedMember<T extends object>(
```

Comment:
If this is only comparing definition, what if we rename this to `compareMemberDefinition`?